options/locale/locale_en-US.ini

```diff
@@ -523,10 +534,11 @@ can_read_info = Read
 can_write_info = Write
 key_state_desc = This key has been used in the last 7 days
 token_state_desc = This token has been used in the last 7 days
+principal_state_desc = This principal has been used in the last 7 days
 show_openid = Show on profile
 hide_openid = Hide from profile
 ssh_disabled = SSH Disabled
-
+ssh_builtin_not_supported = Not supported by builtin SSH
```

Comment:
This seems unused and if it will be used, I'd reword to `Not supported when using the builtin SSH server`

Reply:
good catch, forgot to remove it when I added support, fixed now in new commit